### PR TITLE
Fixed Pings in embeds on mobile & Small changes to README

### PR DESCRIPTION
**Pull request by Mysterium422**
IMG_1697

Should fix issue on mobile where pings don't show up.

Also made some small changes to the README for grammar & tone fixes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,18 +32,18 @@ We will have to do this once:
 After all this, start the bot with `npm run start`.
 
 > ### ⚠ Warning 
-> There is literally no warranty if you self-host Countr, and we will not help you set it up either. If you wish to set the bot up yourself, we expect you have well enough knowledge in Node.js. We still recommend using the original bot.
+> There is no warranty if you self-host Countr, and we will not be helping you set it up either. If you wish to host the bot yourself, we expect you have to have the necessary knowledgee of Node.js. We still recommend using the original bot.
```
Comment (lacuna-exe) [inline README.md:35]: would recommend reverting be helping to help , knowledge has one too many e's.

File: README.md [hunk 2]
```diff
@@ -52,4 +52,4 @@ We use the GNU GPLv3-license.
 
 > You may copy, distribute and modify the software as long as you track changes/dates in source files. Any modifications to or software including (via compiler) GPL-licensed code must also be made available under the GPL along with build & install instructions.
 
-Fetched from [TLDRLegal](https://tldrlegal.com/license/gnu-general-public-license-v3-(gpl-3)), please also read the [license](https://github.com/countr/countr/blob/master/LICENSE) if you plan on using the source code. This is only a short summary. Please also take note of that we are not forced to help you, and we won't help you host it yourself as we do not recommend you doing so.
+Fetched from [TLDRLegal](https://tldrlegal.com/license/gnu-general-public-license-v3-(gpl-3)), please also read the [license](https://github.com/countr/countr/blob/master/LICENSE) if you plan on using the source code. This is only a short summary. Please take note that we will not provide much help, and heavily recommend using the standard bot.
```
Comment (fabianneufang) [inline README.md:55]: We dont provide **any** help

**Review by fabianneufang — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (promise) [reply README.md:55]: sometimes

Comment (fabianneufang) [reply README.md:55]: Yeah sometimes. But we are not going to help setting everything up

**Review by promise — COMMENTED** [review]
have you tested this? doesn't really make a difference for me.

File: README.md (hunk 1 above)
Comment (promise) [reply README.md:35]: eh it looks fine

**Comment by fabianneufang** [comment]
> have you tested this? doesn't really make a difference for me.

For me neither

**Comment by Ta1al** [comment]
### 1. README.md
Disapproval: I would honestly prefer the wording as it was before in README.md (except for the grammar fix on line 25). The older wording sounds like a more down to earth developer.

### 2. src/constants/index.js
This is bug with discord on mobile itself. The only difference between `<@!id>` and `<@id>` is that `<@!id>` displays the nickname in the server, so unnecessary change/s.

**Comment by Mysterium422** [comment]
Thats odd, my bot doesn't have issues on mobile, and that was the only real difference in formatting so I assumed that was the issue. Either way, nicknames should be shown over the tag imo.
Maybe its server members intent or something like that? Not sure what settings you have set on dev app.

Truth be told I only wanted to try forking something and then creating a pull request; its the first time for me xD. I figured that the formatting change would fix things as I thought it did for me.
I still think the Readme I proposed is better (with the fixes that Void proposed)...but I don't even know how to implement those lol. Sorry!

**Comment by Ta1al** [comment]
> Thats odd, my bot doesn't have issues on mobile, and that was the only real difference in formatting so I assumed that was the issue. Either way, nicknames should be shown over the tag imo.
> Maybe its server members intent or something like that? Not sure what settings you have set on dev app.
> 
> Truth be told I only wanted to try forking something and then creating a pull request; its the first time for me xD. I figured that the formatting change would fix things as I thought it did for me.
> I still think the Readme I proposed is better (with the fixes that Void proposed)...but I don't even know how to implement those lol. Sorry!

You made a great first PR! You explained your changes nicely and followed up on comments and change requests. But it really comes down to the original author(s) to approve or disapprove of a PR and you should never get disappointed if the latter happens, at least you tried to help!

Promise gets to decide whether this gets merged or not, but still, a good first PR :)

**Comment by Mysterium422** [comment]
> > Thats odd, my bot doesn't have issues on mobile, and that was the only real difference in formatting so I assumed that was the issue. Either way, nicknames should be shown over the tag imo.
> > Maybe its server members intent or something like that? Not sure what settings you have set on dev app.
> > Truth be told I only wanted to try forking something and then creating a pull request; its the first time for me xD. I figured that the formatting change would fix things as I thought it did for me.
> > I still think the Readme I proposed is better (with the fixes that Void proposed)...but I don't even know how to implement those lol. Sorry!
> 
> You made a great first PR! You explained your changes nicely and followed up on comments and change requests. But it really comes down to the original author(s) to approve or disapprove of a PR and you should never get disappointed if the latter happens, at least you tried to help!
> 
> Promise gets to decide whether this gets merged or not, but still, a good first PR :)

ty :p it wasn't much but was thrilling in a way somehow xD

**Comment by fabianneufang** [comment]
Closing because bug is not fixed

**Pull request closed without merging**